Library/Homebrew/cmd/update.sh

```diff
@@ -406,8 +406,7 @@ This restriction has been made on GitHub's request because updating shallow
 clones is an extremely expensive operation due to the tree layout and traffic of
 Homebrew/homebrew-core and Homebrew/homebrew-cask. We don't do this for you
 automatically to avoid repeatedly performing an expensive unshallow operation in
-CI systems (which should instead be fixed to not use shallow clones). Sorry for
-the inconvenience!
+CI systems (which should use shallow clones). Sorry for the inconvenience!
```

Comment:
> which should use shallow clones

CI shouldn't use shallow clones, CI should use non-shallow clones.

Reply:
so, everyone should use unshallow clones?  why make the distinction between users and CI systems?

Reply:
I'll redact this PR since it is incorrect, but consider this a ➕ 1 on [this comment](https://github.com/Homebrew/discussions/discussions/226#discussioncomment-220932) which accurately describes the experience I had when upgrading

Reply:
I think the messaging was already fixed in https://github.com/Homebrew/brew/pull/9501.